domain/relation/service/relation.go

```diff
@@ -256,9 +278,13 @@ func (s *Service) GetRelationUnitSettings(
 	return nil, errors.NotImplemented
 }
 
-// GetRelationUUIDByID returns the relation uuid based on the relation ID.
+// GetRelationUUIDByID returns the relation UUID based on the relation ID.
+//
+// The following error types can be expected to be returned:
```

Comment:
todo: update the caller to look of this error rather than `errors.NotFound`

https://github.com/juju/juju/blob/68a725e296268952259f1468493bacbd33c4e9d6/apiserver/facades/agent/uniter/uniter.go#L1797